### ISSUE TITLE
Cannot define Entity Type

### ISSUE THREAD

**Issue by SCullman** [2sic/2sxc#1863]
**I'm submitting a ...** 

[x] bug report => search github for a similar issue before submitting 
**...about**   
[x] admin experience UI

**Current behavior**

It is not possible to set a Entity Type

**Expected behavior**
Either the typed in Entity type is respected or 
it is possible to select the Entity Type from a list. 

**Minimal reproduction of the problem with instructions**

1. Create a new App
1. Create new Entitiy: Author (Name:string)
1. Create new Entity Article (Title:string, Author: Entity)
1. Edit field Author and try to set Entity Type

* If you enter the string Author, it will not be saved.
* The List is always empty
![image](https://user-images.githubusercontent.com/5693190/67633068-8373a100-f8ab-11e9-9c81-dba6668157ed.png)

**Setup**  

* **2sxc version(s):** 10.9.1  
* **DNN:** 9.4.1

**Comment by iJungleboy** [comment]
Thanks! we'll prioritize.

**Comment by SCullman** [comment]
FYI: same/ similar error in DNN9.2/ 2sxc 9.43.2 LTS

![image](https://user-images.githubusercontent.com/5693190/67659224-313e8880-f95c-11e9-816e-869d9c8d2354.png)

**Comment by iJungleboy** [comment]
@SCullman We tried to reproduce the issue but ATM it work on my system. When you are configuring the field, do you see a request in F12 like 
```
desktopmodules/2sxc/api/app/auto/query/Eav.Queries.Global.ContentTypes/Default?includeGuid=true
```
This is the request that should get the list of content-types for the dropdown. 

Also, do you pick an item from the list, or just type (which is just searching)?

**Comment by SCullman** [comment]
Ok. Thank you for checking, and I did the same again. I guess you did one more step than I did; I did not add a single instance of that app "confirmed". I did not thought about that because currently I have no views.

Instead, I opened `manage all apps` without selecting the desired app first.. 
  
![image](https://user-images.githubusercontent.com/5693190/67680660-f3f1ef00-f98b-11e9-9b50-18db7145bedc.png)

If I click on the app first, 'configure app' will work as expected. I guess it missing a "context".

A variation of this weird behavior:
If I open `Manage apps` starting from a certain app, for example "Blog", and manage an other app, 
the selection for the entity types are still from the starting app 'Blog', not from the current managed app:  
![image](https://user-images.githubusercontent.com/5693190/67681215-2b14d000-f98d-11e9-99a0-3e6a1c6aa750.png)
